.github/workflows/main.yml

```diff
@@ -7,7 +7,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
-        python: [2.7, 3.5, 3.6, 3.7, 3.8, pypy3]
+        python: [2.7, 3.6, 3.7, 3.8, 3.9, pypy3]
```

Comment:
Removing 3.5 is a breaking change for frozen-flask, no matter the end of life. The policy is to follow whatever flask itself tests against. I'm inclined to releasing a breaking version, which would drop support for obsolete Python versions (opinions, @tswast?), but then let's do it properly. I quickly checked what versions are supported by flask and it seems they might have dropped even the 2.x line 😮 

So my idea would be that we add support for 3.9 and release a minor version. Then we could prepare a version, which drops all the old stuff unsupported by Flask, and only then release a breaking version.